kythe/typescript/indexer.ts

```diff
@@ -230,7 +230,7 @@ class SymbolVNameStore {
   get(symbol: ts.Symbol, ns: TSNamespace, context: Context): VName|undefined {
     if (this.store.has(symbol)) {
       const nsCtx = this.serialize(ns, context);
-      return this.store.get(symbol)!.get(nsCtx);
+      return this.store.get(symbol).get(nsCtx);
```

Comment:
I am surprised by this change.  What tool thinks this is right?

Reply:
sending you a pointer to the downstream analyzer result

Reply:
abandoning this in favor of #4003 then?

Reply:
Yeah I will fix this in g3.